### DISCUSSION TITLE
TP4 - Ejercicio 2 - Ordenamiento II

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#126] [Preguntas y respuestas] [answered]
Una duda. ¿En el punto 2 es si o si un arreglo o un valor entero?

**Comment by martinvilu** [comment, marked as answer]
Cómo es la consigna? Desde el teléfono de complica

**Reply by Elias-Turner** [reply]
La consigna dice:
Implementar una función que, en el ordenamiento de arreglos de números enteros, haga uso de la función.

```C
void intercambia(int* izquierda, int* derecha);
```

Para hacer los movimientos de los valores en el arreglo en lugar de hacer las asignaciones entre las posiciones.

**Reply by martinvilu** [reply]
Un arreglo
